tla/consensus/ccfraft.tla

```diff
@@ -788,18 +788,20 @@ RejectAppendEntriesRequest(i, j, m, logOk) ==
        \/ /\ m.term >= currentTerm[i]
           /\ leadershipState[i] = Follower
           /\ ~logOk
-          /\ LET prevTerm == IF m.prevLogIndex = 0 THEN StartTerm
+          \* raft.h::send_append_entries_response:1348 RAeR FAIL set their term to the term of index
```

Comment:
I wasn't sure what RAeR stood for

Reply:
```suggestion
          \* raft.h::send_append_entries_response:1348 AppendEntriesResponse messages with answer == FAIL set their term to the term of index
```

I'm also unsure about this - I thought RAeR was "Receive Append Entries Request", which is the top-level action we're doing here, but maybe it's actually "_Reject_ Append Entries Request", which is the action we're currently in? But then in other places I thought we used AE for AppendEntries, distinct from AER for AppendEntriesResponse, so maybe AeR isn't Append Entries Request here?